e2e/test/scenarios/admin/settings/license-and-billing.cy.spec.js

```diff
@@ -36,42 +36,40 @@ describe("scenarios > admin > license and billing", () => {
         NO_UPSELL_FEATURE_HEY,
       ]);
 
-      // TODO: remove once endpoint returns values as expected
-      cy.intercept("GET", "/api/ee/billing", {
-        version: "v1",
-        content: [
-          {
-            name: "Plan",
-            value: "Metabase Cloud Pro",
-            format: "string",
-            display: "value",
-          },
-          {
-            name: "Billing frequency",
-            value: "Monthly",
-            format: "string",
-            display: "value",
-          },
-          {
-            name: "Next charge value",
-            value: 500,
-            format: "currency",
-            currency: "USD",
-            display: "value",
-          },
-          {
-            name: "Visit the Metabase store to manage your account and billing preferences.",
-            value: "Manage preferences",
-            format: "string",
-            display: "external-link",
-            link: "https://store.metabase.com/",
-          },
-        ],
-      });
+      const harborMasterConnectedAccount = {
+        email: "ci-admins@metabase.com",
+        first_name: "CI",
+        last_name: "Admins",
+        password: "d4YaINVhIZliV3",
+      };
 
-      cy.visit("/admin/settings/license");
-      cy.findByTestId("billing-info-key-plan").should("exist");
-      cy.findByTestId("license-input").should("exist");
+      // create an admin user who is also connected to our test harbormaster account
+      cy.request("GET", "/api/permissions/group")
+        .then(({ body: groups }) => {
+          const adminGroup = groups.find(g => g.name === "Administrators");
+          return cy
+            .createUserFromRawData(harborMasterConnectedAccount)
+            .then(user => Promise.resolve([adminGroup.id, user]));
+        })
+        .then(([adminGroupId, user]) => {
+          const data = { user_id: user.id, group_id: adminGroupId };
+          return cy
+            .request("POST", "/api/permissions/membership", data)
+            .then(() => Promise.resolve(user));
+        })
+        .then(user => {
+          cy.signOut(); // stop being normal admin user and be store connected admin user
+          return cy.request("POST", "/api/session", {
+            username: user.email,
+            password: harborMasterConnectedAccount.password,
+          });
+        })
+        .then(() => {
+          // core test
+          cy.visit("/admin/settings/license");
+          cy.findByTestId("billing-info-key-plan").should("exist");
+          cy.findByTestId("license-input").should("exist");
+        });
```

Comment:
this .then block is unnecessary (but also not harmful). cypress promises aren't "real" promises.